app/config.py

```
@@ -22,12 +22,12 @@ class Settings(BaseSettings):
     REDIS_URL: str = f"redis://{REDIS_HOST}:{REDIS_PORT}/{REDIS_DB}"
 
     ####Database######
-    DATABASE_USERNAME: str = os.getenv("DATABASE_USERNAME", "username")
-    DATABASE_PASSWORD: str = os.getenv("DATABASE_PASSWORD", "password")
-    DATABASE_HOST: str = os.getenv("DATABASE_HOST", "localhost")
-    DATABASE_PORT: int = int(os.getenv("DATABASE_PORT", "5432"))
-    DATABASE_NAME: str = os.getenv("DATABASE_NAME", "wwa_db")
-    DATABASE_DBAPI: str = os.getenv("DATABASE_DBAPI", "postgresql+asyncpg")
+    DATABASE_USERNAME: str = "username"
+    DATABASE_PASSWORD: str = "password"
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to: "DATABASE_PASSWORD"

(S105)